build.gradle

```diff
@@ -370,3 +380,7 @@ if (CacheRedirector.enabled) {
         nodeJsExtension.nodeDownloadBaseUrl = CacheRedirector.maybeRedirect(nodeJsExtension.nodeDownloadBaseUrl)
     }
 }
+
+extensions.findByType(org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootExtension.class).with {
```

Comment:
This is NodeJs version that supports needed wasm. It can be removed when default nodejs version in kotlin gradle plugin will be hight to run Kotlin/Wasm output.

Reply:
Then it's worth dropping a comment right there.

Reply:
Fixed

Reply:
Do we have a ticket for that?
For now, comment is still non-actionable for other maintainers

Reply:
A compartible nodejs Its is very dependent from the compiler version (even on build) and will be changed over time.
You are asking about a some kind of compatibility ticket?

Reply:
Anything that will hint to the maintainer what to do.

Consider this: we have some non-trivial configuration that fixes the node version; the other maintainer approaches this and tries to understand whether they can safely remove it or update (or update in the whole project) it, and there are no hints whatsoever.

Probably something like this: `// Drop this configuration when the node JS version in KGP is at least X, check it here: LINK`